### ISSUE TITLE
Inconsistency between function descriptions and example sketch

### ISSUE THREAD

**Issue by Araris** [burakcanturk/HLK_LD2450_Library#1]
Hi, [burakcanturk](https://github.com/burakcanturk)
Thank you for the work you have done to create the library.
My problem is that in the example sketch, function calls are made with a parameter (target number), and there is no mention of multi-targeting in the library. 
Is it possible to add multitargeting ?
Tekrar teşekkürler.

**Comment by burakcanturk** [comment]
You're welcome. Do you have HLK-LD2450's PC application that you can see
the values in display? İf you don't, I can send the application. This
application can read Multi values that you want.

12 Şub 2024 Pzt 16:44 tarihinde Araris ***@***.***> şunu
yazdı:

**Comment by Araris** [comment]
Thank you for quick response.
Yes, I have the mentioned program and I saw the necessary data on the display.
But my goal is to connect the HLK-LD2450 to the ESP32 and receive data into my program.
As far as I understand, you do not have a multi-targeting version of the library, perhaps you should fix the example sketch.
Well, this is an opportunity for me to use my own head ))).

**Comment by rikki78** [comment]
Same question here 😃 
Can you update the library? From the examples it looks like the library is capable of multi targeting but only one is supported.

**Comment by mianos** [comment]
mine, for the esp32 has multiple targets

**Comment by rikki78** [comment]
> mine, for the esp32 has multiple targets

You mean this one? https://github.com/mianos/radar2

**Comment by mianos** [comment]
Yes, but that is a derivative of this one that has state machine based drivers for all their radars: https://github.com/mianos/hk-heltec-radar/tree/main/src

The doorbell2 just has the 2450. They both publish to mqtt. 
Both are running all around me as I sit here. If you have any issues.getting them going give me a shout.

**Comment by Araris** [comment]
Hi, rikki78, maybe this will help you :
[TestHLK-LD2450.zip](https://github.com/burakcanturk/HLK_LD2450_Library/files/14897388/TestHLK-LD2450.zip)
